Source/ModelGroups/Instancing.cs

```diff
@@ -357,6 +357,20 @@ void AddAnimation(List<Animation> animations, List<Node> nodes, AnimationSampler
                     properties.Add(new Property(PropertyName.Difference, "The two nodes have different translations."));
                 }, (model) => { model.Camera = distantCamera; }),
                 CreateModel((properties, nodes, animations) =>
+                {
+                    var meshPrimitive = MeshPrimitive.CreateSinglePlane();
+                    meshPrimitive.Material = CreateMaterial(CreateTextureInfo(baseColorImageA));
+
+                    AddMeshPrimitivesToMultipleNodes(nodes, meshPrimitive, meshPrimitive);
+                    nodes[1].Mesh = nodes[0].Mesh;
+
+                    // Adds just the node containing the negative scale.
+                    nodes[0].Scale = new Vector3(-1.0f, -1.0f, 1.0f);
```

Comment:
Flipping two axes doesn't test properly. The transform matrix will end up with a positive determinant which isn't going to test this case properly. We just need to flip one axis.

```suggestion
                    nodes[0].Scale = new Vector3(1.0f, -1.0f, 1.0f);
```

Reply:
Got it. Thanks Gary.